resources/js/Components/Table/EntryTable/EntryTable.tsx

```diff
@@ -0,0 +1,114 @@
+import {useForm} from '@inertiajs/inertia-react'
+import DeleteIcon from '@mui/icons-material/Delete'
+import EditIcon from '@mui/icons-material/Edit'
+import {Box, Chip, IconButton, Stack, Tooltip} from '@mui/material'
+import {DataTable, DeleteModal} from 'Components'
+import React, {useState} from 'react'
+import {EntriesData} from 'Types/entries-data'
+import route from 'ziggy-js'
+
+type EntryPropType = {
+    data: EntriesData[]
+}
+
+type FormDataType = {
+    entries: number[]
+}
+
+const EntryTable = ({data}: EntryPropType) => {
+    const [selected, setSelected] = useState<number[]>([])
+    const [isDeleteModalOpen, setIsDeleteModalOpen] = useState<boolean>(false)
+    const {setData, delete: destroy} = useForm<FormDataType>({entries: []})
+
+    const columns = [
+        {
+            property: 'date_played',
+            title: 'Date',
+        },
+        {
+            property: 'adventure',
+            title: 'Adventure Title',
+            render: (value: any) => (
+                <Chip label={value ? value.title : 'N/A'} variant='outlined' />
+            ),
+        },
+        {
+            property: 'session',
+            title: 'Session',
+        },
+        {
+            property: 'reward',
+            title: 'Reward',
+        },
+        {
+            property: 'items.name',
+            title: 'Magic Items',
+            render: (value: any) => (
+                <Chip aria-label='item' label={value.items.name} variant='outlined' />
```

Comment:
i don't this this works. `value.items` is an array, so wouldn't you have to iterate over each item in the array to create a chip for each item? 
Since you already have approval, you can go ahead and just open a bug that references this comment, and fix it this sprint.

Reply:
@AnnieTran13

Reply:
I made a new issue that references this comment.